window/src/os/windows/window.rs

```diff
@@ -21,62 +21,87 @@ use std::any::Any;
 use std::cell::RefCell;
 use std::collections::HashMap;
 use std::convert::TryInto;
-use std::ffi::OsString;
+use std::ffi::{c_short, c_void, OsString};
+use std::hash::Hash;
 use std::io::{self, Error as IoError};
 use std::os::windows::ffi::OsStringExt;
 use std::path::PathBuf;
-use std::ptr::{null, null_mut};
 use std::rc::Rc;
 use std::sync::Mutex;
 use wezterm_color_types::LinearRgba;
 use wezterm_font::FontConfiguration;
 use wezterm_input_types::KeyboardLedStatus;
-use winapi::shared::minwindef::*;
-use winapi::shared::ntdef::*;
-use winapi::shared::windef::*;
-use winapi::shared::winerror::S_OK;
-use winapi::um::imm::*;
-use winapi::um::libloaderapi::GetModuleHandleW;
-use winapi::um::shellapi::{DragAcceptFiles, DragFinish, DragQueryFileW, HDROP};
-use winapi::um::sysinfoapi::{GetTickCount, GetVersionExW};
-use winapi::um::uxtheme::{
+use windows::core::PCWSTR;
+use windows::Win32::Foundation::*;
+use windows::Win32::Globalization::HIMC;
+use windows::Win32::Graphics::Dwm::{
+    DwmEnableBlurBehindWindow, DWM_BB_BLURREGION, DWM_BB_ENABLE, DWM_BLURBEHIND,
+};
+use windows::Win32::Graphics::Gdi::{
+    BeginPaint, ClientToScreen, CreateRectRgn, DeleteObject, EndPaint, GetDC, GetMonitorInfoW,
+    InvalidateRect, MonitorFromWindow, ReleaseDC, ScreenToClient, HBRUSH, HDC, LOGFONTW,
+    MONITORINFO, MONITORINFOEXW, MONITOR_DEFAULTTONEAREST, MONITOR_DEFAULTTOPRIMARY, PAINTSTRUCT,
+};
+use windows::Win32::System::LibraryLoader::GetModuleHandleW;
+use windows::Win32::System::SystemInformation::{GetTickCount, GetVersionExW, OSVERSIONINFOW};
+use windows::Win32::System::SystemServices::{
+    MK_CONTROL, MK_LBUTTON, MK_MBUTTON, MK_RBUTTON, MK_SHIFT,
+};
+use windows::Win32::UI::Controls::{
     CloseThemeData, GetThemeFont, GetThemeSysFont, OpenThemeData, SetWindowTheme,
+    THEME_PROPERTY_SYMBOL_ID,
+};
+use windows::Win32::UI::HiDpi::{GetDpiForWindow, GetSystemMetricsForDpi};
+use windows::Win32::UI::Input::Ime::{
+    ImmGetCompositionStringW, ImmGetContext, ImmReleaseContext, ImmSetCandidateWindow,
+    ImmSetCompositionWindow, CANDIDATEFORM, CFS_EXCLUDE, CFS_POINT, COMPOSITIONFORM, GCS_COMPSTR,
+    GCS_RESULTSTR, IME_COMPOSITION_STRING, ISC_SHOWUICOMPOSITIONWINDOW,
+};
+use windows::Win32::UI::Input::KeyboardAndMouse::{
+    GetFocus, GetKeyState, GetKeyboardLayout, GetKeyboardState, MapVirtualKeyW, ReleaseCapture,
+    SendInput, SetCapture, SetKeyboardState, ToUnicode, TrackMouseEvent, INPUT, INPUT_KEYBOARD,
+    KEYBDINPUT, KEYEVENTF_EXTENDEDKEY, KEYEVENTF_KEYUP, MAPVK_VK_TO_VSC, TME_LEAVE, TME_NONCLIENT,
+    TRACKMOUSEEVENT, VK_CAPITAL, VK_CONTROL, VK_DECIMAL, VK_LCONTROL, VK_LMENU, VK_LSHIFT, VK_LWIN,
+    VK_MENU, VK_NUMLOCK, VK_PACKET, VK_PROCESSKEY, VK_RCONTROL, VK_RMENU, VK_RSHIFT, VK_RWIN,
+    VK_SHIFT,
+};
+use windows::Win32::UI::Shell::{DragAcceptFiles, DragFinish, DragQueryFileW, HDROP};
+use windows::Win32::UI::TextServices::HKL;
+use windows::Win32::UI::WindowsAndMessaging::{
```

Comment:
This is big enough that it is starting to dominate the file, so I think that a wildcard import would be totally fine.

```suggestion
use windows::Win32::UI::WindowsAndMessaging::*;
```